.travis.yml

```diff
@@ -56,3 +56,6 @@ jobs:
     - stage: test
       script:
       - make golint
+    - stage: test
```

Comment:
Looks like this change is unrelated

Reply:
yeah, I would remove this, added this stage just to check the CI running this script `make release-snapshot`

Reply:
Got it 👍🏼

Reply:
have removed this change.